shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-env/src/test/java/org/apache/shardingsphere/test/integration/env/container/atomic/adapter/config/AdaptorContainerConfiguration.java

```diff
@@ -25,8 +25,8 @@
 /**
  * Adaptor container configuration.
  */
-@RequiredArgsConstructor
 @Getter
+@RequiredArgsConstructor
```

Comment:
Please do not change the original codes if unnecessary.

Reply:
done